privaterelay/settings.py

```diff
@@ -201,9 +201,15 @@
 DAYS_PER_BILLING_CYCLE = config("DAYS_PER_BILLING_CYCLE", 30, cast=int)
 MAX_DAYS_IN_MONTH = 31
 IQ_ENABLED = config("IQ_ENABLED", False, cast=bool)
+IQ_FOR_VERIFICATION = config("IQ_FOR_VERIFICATION", False, cast=bool)
 IQ_FOR_NEW_NUMBERS = config("IQ_FOR_NEW_NUMBERS", False, cast=bool)
+IQ_MAIN_NUMBER = config("IQ_MAIN_NUMBER", "")
 IQ_OUTBOUND_API_KEY = config("IQ_OUTBOUND_API_KEY", None)
 IQ_INBOUND_API_KEY = config("IQ_INBOUND_API_KEY", "")
+IQ_MESSAGE_PATH = config(
```

Comment:
When would we customize `IQ_MESSAGE_PATH`? Putting it in settings suggests to me that it may change in different deployments, like stage vs production

Reply:
I found myself duplicating this code in a few places (models, api view, tests):
```python
iq_publish_message_url = f"{IQ_MESSAGE_API_ORIGIN}/msgbroker/rest/publishMessages"
```

So, I wanted to move it "up" into a module that is easy to import in all those places.

But that's weird ... in my local branch I have:
```python
  IQ_MESSAGE_API_ORIGIN = config(
      "IQ_MESSAGE_API_ROOT", "https://messagebroker.inteliquent.com"
  )
  IQ_MESSAGE_PATH = "/msgbroker/rest/publishMessages"
  IQ_PUBLISH_MESSAGE_URL = f"{IQ_MESSAGE_API_ORIGIN}{IQ_MESSAGE_PATH}"
```

I.e., I don't pull the `IQ_MESSAGE_PATH` from an env var - I just hard-code it. If this looks okay, maybe I just need to figure out why this PR seems to have some changes I didn't mean for it to have, and it's misisng some changes that it should have.